README.md

```diff
@@ -63,24 +63,75 @@ def test_full_cycle(project, remote_orchestrator):
     service_instance.delete()
 
 ```
-## Options
-
-The following options are available.
-
- * `--lsm_host` remote orchestrator to use for the remote_orchestrator fixture, overrides INMANTA_LSM_HOST
- * `--lsm_user` username to use to ssh to the remote orchestrator, overrides INMANTA_LSM_USER
- * `--lsm_port` port to use to ssh to the remote orchestrator, overrides INMANTA_LSM_PORT
- * `--lsm_environment` the environment to use on the remote server (it is created if it doesn't exist), overrides INMANTA_LSM_ENVIRONMENT
- * `--lsm_noclean` Don't cleanup the orchestrator after last test (for debugging purposes)
- * `--lsm_ssl` Connect to the remote orchestrator using SSL/TLS, overrides INMANTA_LSM_SSL
- * `--lsm_token` The token used to authenticate to the remote orchestrator when authentication is enabled, overrides INMANTA_LSM_TOKEN
- * `--lsm_ca_cert` The path to the CA certificate file used to authenticate the remote orchestrator, overrides INMANTA_LSM_CA_CERT
- 
-## Environment variables
+## Options and environment variables
 
-The following environment variables are available:
+The following options are available, each with a corresponding environment variable.
 
- * `INMANTA_LSM_MODULE_CONSTRAINTS` : semi-colon separated list of constraints e.g : `"lsm~=2.12.0"`
+
+```
+pytest-inmanta-lsm:
+  --lsm-ca-cert=LSM_CA_CERT
+                        The path to the CA certificate file used to authenticate the remote
+                        orchestrator. (overrides INMANTA_LSM_CA_CERT)
+  --lsm-container-env   If set to true, expect the orchestrator to be running in a container
+                        without systemd.  It then assumes that all environment variables required
+                        to install the modules are loaded into each ssh session automatically.
+                        (overrides INMANTA_LSM_CONTAINER_ENV)
+  --lsm-doc-orch        If set, the fixtures will deploy and orchestrator on the host, using docker
```

Comment:
I don't know if doc is a good short hand, as it usually refers to documentation. 

(but can't think of anything better)

Reply:
I changed the naming, no mode doc or docker, I used `ctr` (short for container), I also removed `orch/orchestrator` as we already have lsm in there